core/src/raw/http_util/client.rs

```diff
@@ -78,6 +90,41 @@ impl HttpClient {
 
     /// Fetch a request in async way.
     pub async fn fetch(&self, req: Request<Buffer>) -> Result<Response<HttpBody>> {
+        self.fetcher.fetch(req).await
+    }
+}
+
+pub trait HttpFetch: Send + Sync + Unpin + 'static {
```

Comment:
Please add comments to this trait. This trait is exposed to users, and we expect that users should never interact with `HttpFetchDyn`.